lms/templates/courseware/submission_history.html

```diff
@@ -1,11 +1,11 @@
 <%page expression_filter="h"/>
-<% import json  %>
+<% import json, pytz  %>
 <h3>${username} > ${course_id} > ${location}</h3>
 
 % for i, (entry, score) in enumerate(zip(history_entries, scores)):
 <hr/>
 <div>
-<b>#${len(history_entries) - i}</b>: ${entry.updated} UTC</br>
+<b>#${len(history_entries) - i}</b>: ${entry.updated.astimezone(pytz.timezone(settings.TIME_ZONE))} (${settings.TIME_ZONE} time)</br>
```

Comment:
"time" should be translated, and the string should be formatted with wildcards instead of being concatenated together. And then there is the matter of settings.TIME_ZONE being an English string as well. Is there any way to just get the abbreviation for the time zone and append that (without also adding "time")? I suppose one could argue that it is still not translated, but those are more widely understood abbreviations.

I think you can use localize to get this information. http://stackoverflow.com/questions/5946499/how-to-get-the-common-name-for-a-pytz-timezone-eg-est-edt-for-america-new-york